ta/aes_perf/include/user_ta_header_defines.h

```diff
@@ -32,7 +32,10 @@
 
 #define TA_UUID TA_AES_PERF_UUID
 
-#define TA_FLAGS		(TA_FLAG_USER_MODE | TA_FLAG_EXEC_DDR)
+#define TA_FLAGS                (TA_FLAG_USER_MODE | TA_FLAG_EXEC_DDR | \
+				TA_FLAG_SECURE_DATA_PATH | \
+				TA_FLAG_CACHE_MAINTENANCE)
```

Comment:
That's OK. Sorry, I misunderstood the meaning of the flag when I made this comment (reading it as "I request SDP support" when it really means "I want to be allowed to access SDP buffers (if SDP is supported)").